src/components/Dropdown/index.tsx

```diff
@@ -1,35 +1,46 @@
 import { Dispatch, SetStateAction } from "react";
 import { useDispatch } from "react-redux";
-import { setIsLoggedIn } from "@/src/store/global";
-
-import { notifyError, notifySuccess } from '../../utils/toast';
-import { DropdownPresentation } from "./Presentation";
+import { setIsLoggedIn, setUserData } from "@/src/store/global";
+import { useLogoutUserMutation } from "@/src/services/logoutApi";
 
 import { DROPDOWN_LINKS } from "./DropdownConstants";
 
-import styles from "./dropdown.module.css";
-import { useLogoutUserMutation } from "@/src/services/logoutApi";
+import { notifyError, notifySuccess } from "../../utils/toast";
+import { DropdownPresentation } from "./Presentation";
```

Comment:
Why are you not using absolute(@) import here?

Reply:
This is not a test file and you are already using absolute import for some components

Reply:
Yes i know still test cases fails because jest is not configure here for absolute import i can show you a demo if you want